pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
i think you still miss the point, i fully understand that... but thats not how you wrote the whole other calls to one that pass an argument that is none. if you then set the address to $pc everything blows up ;-)